Gruntfile.js

```diff
@@ -6,18 +6,18 @@ module.exports = function(grunt) {
     var dir =  'embeds/' + (grunt.option('folderName') ? grunt.option('folderName') : '');
     var scss = 'embeds/' + (grunt.option('folderName') ? grunt.option('folderName') + '/*.scss' : '**/*.scss');
     var html = 'embeds/' + (grunt.option('folderName') ? grunt.option('folderName') + '/*.html' : '**/*.html');
-    var source = 'embeds/' + (grunt.option('folderName') ? grunt.option('folderName')+ '/source/*' : '**/source/*');
+    var source = 'embeds/' + (grunt.option('folderName') ? grunt.option('folderName')+ '/_source/*' : '**/source/*');
     var remoteDir = 'thrashers/' + (grunt.option('folderName') ? grunt.option('folderName') : '');
 
     grunt.initConfig({
         watch: {
             local: {
                 files: [scss, html, source],
-                tasks: ['sass', 'compile']
+                tasks: ['sass', 'hash', 'compile']
             },
             remote: {
                 files: [scss, html, source],
-                tasks: ['sass', 'compile', 'aws_s3']
+                tasks: ['sass', 'hash', 'compile', 'aws_s3']
```

Comment:
No because otherwise the references (added in the compile task) won't get updated to the latest hash. Also then they won't get uploaded until the next time it is run.